saturn-rapids/.env_deps

```diff
@@ -1,3 +1,3 @@
-VERSION=2021.07.26-0
-SATURNBASE_GPU_IMAGE=saturncloud/saturnbase-gpu:2021.06.23
-IMAGE=saturncloud/saturn-rapids:2021.07.26-0
+VERSION=2021.08.09-0
+SATURNBASE_GPU_IMAGE=saturncloud/saturnbase-gpu:2021.06.23-1
```

Comment:
I think this should point to saturnbase-gpu-11.2

Reply:
Making that change here wouldn't have any effect, anyway (builds don't use `.env_deps` - they overwrite them) - if that's a change we want to make, it'd need to be made in the release images config (which I won't link to here - this is a public repo).

Reply:
I updated the .env_deps for completeness but I won't worry about it then since it's not relevant